main.py

```diff
@@ -1,10 +1,46 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        data = json.load(file)
+
+    for nickname, player_data in data.items():
+        race_data = player_data.get("race")
+        guild_data = player_data.get("guild")
+
+        race, created = Race.objects.get_or_create(
+            name=race_data["name"],
+            defaults={"description": race_data["description"]}
+        )
+
+        if created:
+            for skill_data in race_data.get("skills"):
+                Skill.objects.create(
+                    name=skill_data["name"],
+                    bonus=skill_data["bonus"],
+                    race=race
+                )
+
+        guild = None
+        if guild_data:
+            guild, created = Guild.objects.get_or_create(
```

Comment:
Since you do not use `created` variable here and on line 35, you could omit it with `_`